spec/fixtures/vcr_cassettes/accept_book_to_nomis_enabled.yml

```diff
@@ -0,0 +1,394 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://noms-api-dev.dsd.io/nomisapi/lookup/active_offender?date_of_birth=1979-04-23&noms_id=A1475AE
```

Comment:
@StupidCodeFactory should the references to the api host name be extracted out to an env var?

Reply:
not really, I can't see the value of it now. It's set in the config/environments/test.rb